src/components/home/modals/NewToken.vue

```diff
@@ -0,0 +1,954 @@
+                }
+                apiSchema = dataInfo
+            } else if (walletType === 'coldWallet') {
+                const coldDataInfo = {
+                    contract: this.support === false ? CONTRACT : CONTRACT_WITH_SPLIT,
+                    senderPublicKey: this.coldAddresses[this.coldAddress].publicKey,
+                    fee: TOKEN_FEE * VSYS_PRECISION,
+                    feeScale: FEE_SCALE,
+                    timestamp: this.dataObject.timestamp,
+                    initData: base58.encode(transaction.prepareCreate(BigNumber(this.coldAmount).multipliedBy(BigNumber(Math.pow(10, this.unity))), BigNumber(Math.pow(10, this.unity)), this.coldTokenDescription)[0]),
+                    description: this.coldContractDescription,
+                    signature: this.coldSignature
+                }
+                apiSchema = coldDataInfo
+            }
+            const url = NODE_IP + '/contract/broadcast/register'
+            this.$http.post(url, apiSchema).then(response => {
+                if (walletType === 'hotWallet') {
+                    this.pageId++
+                } else {
+                    this.coldPageId++
+                }
+                this.tokenId = transaction.contractIDToTokenID(response.body.contractId)
+            }, response => {
+                this.sendError = true
+            })
+            this.$emit('endSendSignal')
+        },
+        nextPage: function() {
+            this.sendError = false
+            this.timeStamp = Date.now() * 1e6
+            this.hasConfirmed = false
+            this.pageId++
+        },
+        coldNextPage: function() {
+            this.sendError = false
+            this.coldPageId++
+        },
+        prevPage: function() {
+            this.sendError = false
+            if (this.pageId === 1) {
+                this.$refs.newTokenModal.hide()
+            } else {
+                this.pageId--
+            }
+        },
+        coldPrevPage: function() {
+            this.sendError = false
+            if (this.coldPageId === 1) {
+                this.$refs.newTokenModal.hide()
+            } else {
+                this.coldPageId--
+            }
+        },
+        resetPage: function() {
+            this.opc = ''
+            this.unity = 8
+            this.qrTotalPage = 1
+            this.qrArray = new Array(0)
+            this.recipient = ''
+            this.amount = BigNumber(0)
+            this.attachment = ''
+            this.pageId = 1
+            this.coldRecipient = ''
+            this.coldAmount = BigNumber(0)
+            this.coldAttachment = ''
```

Comment:
what is the usage of this.attachment = '' and this.coldAttachment = ''?
what is the difference between attachment and contractDescription?